### ISSUE TITLE
Mousemove event while dragging a sortable item

### ISSUE THREAD

**Issue by bribar** [uikit/uikit#3998]
I apologize if posting this here isn't appropriate. I have tried the other suggested help resources first. Discord's barely active it seems. Stackoverflow (https://stackoverflow.com/questions/58240623/uikit-sortable-mouse-position-while-dragging) hasn't provided anything. Maybe this is a feature request?

Anyways, I have tried so many different event listeners. https://jsfiddle.net/bribar/3kh9b5xv

I have looked through the sortable.js and didn't notice anything that would prevent this.

Simply put, I just need to know the mouse position while dragging.

Is this not possible? Or is this a feature? Or am I losing it and missing something?

Thanks for any direction you have!

**Comment by bribar** [comment]
Arrrrrgh! I found it. Something so simple of course. UIkit has pointer-events set to none on .uk-sortable-drag.

`.uk-sortable-drag{
pointer-events: inherit !important;
}`

**Comment by bribar** [comment]
Although I found the reason I couldn't listen to events, changing this css breaks the sortable items. The placeholders don't shift around and mouse up isn't keeping the new sort positions.

Is there any other way?

**Comment by janschoenherr** [comment]
Not sure what you need precisely, but you could register the `pointermove` event on the document. That's what UIkit itself does.

**Comment by bribar** [comment]
WTH! Sorry @janschoenherr to waste your time. That was one of the first events I tried and it wasn't outputting to the console, even had it in my fiddle. Maybe I was starring at my project too long and there was a simple typo or something. Man I wasted too much time on this, haha. Thanks!
